src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -16,13 +16,9 @@
 
 import requests
 
-try:
-    from urllib.parse import urlparse
-except ImportError:
-    from urlparse import urlparse  # pylint: disable=import-error
-
+from urllib.parse import urlparse
 # the urlopen is imported for automation purpose
-from six.moves.urllib.request import urlopen  # noqa, pylint: disable=import-error,unused-import,ungrouped-imports
+from urllib.request import urlopen  # noqa, pylint: disable=import-error,unused-import,ungrouped-imports
```

Comment:
`urlopen` is never used by `custom.py` but is patched by test

https://github.com/Azure/azure-cli/blob/87004321ec280e092fc41febf4ada3b58f01f388/src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_image.py#L30

What is exactly "automation purpose"?

Reply:
This is because in the previous PR https://github.com/Azure/azure-cli/pull/5328, the reference of `urlopen` in the method `load_images_from_aliases_doc()` was modified to use `requests`, but the mock path in `test_read_images_from_alias_doc()` was not synchronously modified.
![Screenshot 2021-09-14 172259](https://user-images.githubusercontent.com/16612065/133232097-efbe3baa-f06d-4ef1-a6a4-dcd455f6211a.png)

Therefore, the import of `from six.moves.urllib.request import urlopen` can be deleted, and we just need to change the mock path to `@mock.patch(azure.cli.command_modules.vm.custom.requests, autospec=True)`

Reply:
We can do this in a separate PR if time permits.